docs/zeropi/readme.md

```diff
@@ -0,0 +1,549 @@
+
+**Wiring Guide** </br>
+
+| NanoPi (pin) | NanoPi (signal) | USB to Serial |
+|:------------:|:---------------:|:-------------:|
+|      1       |    GND          |     black     |
+|      2       |    VDD 5V       |     red       |
+|      3       |    UART TXD0    |     white     |
+|      4       |    UART RXD0    |     green     |
+
+
+**Console Setup** </br>
+
+```zsh
+# use dmesg to verify the tty for your device
+% sudo dmesg -T --follow
+
+# for example, ttyUSB0
+usb 1-3.2: new full-speed USB device number 7 using xhci_hcd
+usb 1-3.2: New USB device found, idVendor=10c4, idProduct=ea60, bcdDevice= 1.00
+usb 1-3.2: New USB device strings: Mfr=1, Product=2, SerialNumber=3
+usb 1-3.2: Product: CP2102 USB to UART Bridge Controller
+usb 1-3.2: Manufacturer: Silicon Labs
+usb 1-3.2: SerialNumber: 0001
+cp210x 1-3.2:1.0: cp210x converter detected
+usb 1-3.2: cp210x converter now attached to ttyUSB0
+
+# if dmesg lists a new USB device but there is no message about tty attached,
+# reboot your host machine with the USB serial cable attached and check again
+
+# install and use picocom to connect to the ZeroPi via /dev/<ttyUSB>
+% sudo pacman -S picocom
+% picocom --baud 115200 /dev/ttyUSB0
+picocom v3.1
+
+port is        : /dev/ttyUSB0
+flowcontrol    : none
+baudrate is    : 115200
+parity is      : none
+databits are   : 8
+stopbits are   : 1
+escape is      : C-a
+local echo is  : no
+noinit is      : no
+noreset is     : no
+hangup is      : no
+nolock is      : no
+send_cmd is    : sz -vv
+receive_cmd is : rz -vv -E
+imap is        : 
+omap is        : 
+emap is        : crcrlf,delbs,
+logfile is     : none
+initstring     : none
+exit_after is  : not set
+exit is        : no
+
+Type [C-a] [C-h] to see available commands
+Terminal ready
+
+# if the ZeroPi is already booted up, hit <enter> and you should see the login prompt.
+
+alarm login: 
+
+```
+
+----------------------------------------------------------------------------------------
+
+### OS Setup </br>
+
+```zsh
+# verify ethernet/dhcp; for example, here eth0 is down but eth1 is up and working
+% ip link
+1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN mode DEFAULT group default qlen 1000
+    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
+2: eth0: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc mq state DOWN mode DEFAULT group default qlen 1000
+    link/ether 7e:41:60:87:ce:1f brd ff:ff:ff:ff:ff:ff
+3: eth1: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc fq_codel state UP mode DEFAULT group default qlen 1000
+    link/ether 00:0e:c6:a4:bb:76 brd ff:ff:ff:ff:ff:ff
+
+# you might need to start dhcp manually (as root)
+% systemctl start dhcpcd.service
+
+# init the pacman keyring
+% pacman-key --init
+% pacman-key --populate archlinuxarm
+
+# install network manager, enable/start, and then stop dhcpcd.service
+% pacman -S networkmanager
+% systemctl stop dhcpcd.service
```

Comment:
This will stop `dhcpcd.service` from running on the current boot, but it will start again on the next reboot cycle. You may have meant this instead:
```bash
systemctl disable --now dhcpcd.service
```

Reply:
removed this section completely - with the new dtb, it all works... like MAGIC.